docs/best_practices/tool_xml.rst

```diff
@@ -135,6 +135,9 @@ escaping (``&amp;``).
 
 `Wikipedia has more on CDATA <http://en.wikipedia.org/wiki/CDATA>`__
 
+All Cheetah variables for text parameters, input and output files must be
+single-quoted, e.g. ``'${var_name}'``.
```

Comment:
I changed to single quotes on purpose because the command is passed to the shell, so if the parameter contains e.g. $foo, with double quotes it will be interpreted as an environment variable.